Builder/ElasticaDatagridBuilder.php

```diff
@@ -148,6 +153,23 @@ public function isSmart(AdminInterface $admin, array $values = array())
             }
 
             if (!in_array($key, $mappedFieldNames)) {
+                /*
+                 * We are in the case where a field is used as filter
+                 * without being mapped in elastic search.
+                 * An ugly case would be to have a custom field used in the filter
```

Comment:
It's the case where you define a field in a filter that is not mapped in ElasticSearch but neither in the model manager.